Makefile.am

```diff
@@ -32,6 +32,10 @@ OSX_PLIST=$(top_builddir)/share/qt/Info.plist #not installed
 OSX_QT_TRANSLATIONS = da,de,es,hu,ru,uk,zh_CN,zh_TW
 
 DIST_DOCS = $(wildcard doc/*.md) $(wildcard doc/release-notes/*.md)
+DIST_CONTRIB = $(wildcard $(top_srcdir)/contrib/*.bash-completion) \
+	       $(top_srcdir)/contrib/debian \
```

Comment:
Now that the manpages have been moved out of `debian`, is there still a reason to include the `debian` directory here?